packages/general/src/util/Error.ts

```diff
@@ -12,6 +12,13 @@ function considerAsError(error: unknown): error is Error {
     return (error as Error).message !== undefined;
 }
 
+export function asError(e: any, prefixText?: string): Error {
```

Comment:
I think the prefix text here is confusing since it only applies if the cause is not an error.  Either the caller wants to add a prefix, in which case we should do it unconditionally, or it gets added at a higher level, in which case we should leave the message as whatever non-error was thrown

So I'd say either take it out or make it apply unconditionally

Reply:
removed